plugins/hypervisors/xenserver/src/com/cloud/hypervisor/xenserver/resource/CitrixResourceBase.java

```diff
@@ -4251,6 +4257,34 @@ public void rebootVM(final Connection conn, final VM vm, final String vmName) th
         }
     }
 
+    protected String skipOrRemoveSR(Connection conn, SR sr) {
```

Comment:
@DaanHoogland I thought that your point was with the fact of being a protected method (not with the fact of being self-explanatory).

Sorry, but when I said, "Is this method self-explanatory?" I meant about the name of it, which I think is self-explanatory (says exactly what it does). Although a documentation could (not should) be applied.